src/main/java/org/quantumbadger/redreader/http/HTTPBackend.java

```diff
@@ -92,16 +92,10 @@ public static String encodeList(final List<PostField> fields) {
 	}
 
 	interface Listener {
+		void onError(@CacheRequest.RequestFailureType int failureType, Throwable exception, Integer httpStatus);
 
-		void onError(RequestFailureType failureType, Throwable exception, Integer httpStatus);
-
-		void onSuccess(
-				String mimetype,
-				Long bodyBytes,
-				InputStream body);
+		void onSuccess(String mimetype, Long bodyBytes, InputStream body);
```

Comment:
Sorry about that, I'm so used to pressing the reformat shortcut in Android Studio. Do you want to keep it in or get rid of it? It'd take a long time to change back again, as before it was automatically done, but I can if need be. I don't imagine it would be too hard for when people merge as the conflicts would be on lines which are basically the same, which would be easy to decide.